### DISCUSSION TITLE
Norton Threat name: Trojan.BazarLoadFull Path: rustdesk-1.2.3-x86_64.exe

### DISCUSSION THREAD

**Discussion by Boxieeee** [rustdesk/rustdesk#7072] [General]
Filename: rustdesk-1.2.3-x86_64.exe
Threat name: Trojan.BazarLoadFull Path: C:\Users\tarun\OneDrive\Downloads\Programs\rustdesk-1.2.3-x86_64.exe

____________________________

____________________________

On computers as of 
2/6/2024 at 5:36:52 PM

Last Used 
2/6/2024 at 5:40:16 PM

Startup Item 
No
Launched 
No
Threat type: Heuristic Virus. Detection of a threat based on malware heuristics.

____________________________

rustdesk-1.2.3-x86_64.exeThreat name: Trojan.BazarLoad
Locate

Many Users
Tens of thousands of users in the Norton Community have used this file.

Mature
This file was released 4 months  ago.

High
This file risk is high.

____________________________

Source: External Media

Source File: 
rustdesk-1.2.3-x86_64.exe

____________________________

**Comment by Mendistern** [comment]
Got the same notification. Does this mean my computer was infected for 4 months?

**Reply by Boxieeee** [reply]
Dunno, but  i think Norton just included this in their scan engine ,because it happened today.

**Comment by twobnt** [comment]
I'm thinkin it is a false positiv finding

https://www.virustotal.com/gui/file/23b661d7bc171cd500d5096456905283ffe06479582b62d3bd5066633935d43e?nocache=1

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/2485
